shared/common-adapters/back-button.desktop.js

```diff
@@ -19,8 +19,8 @@ export default class BackButton extends Component {
   render () {
     return (
       <div style={{...styles.container, ...this.props.style}} onClick={e => this.onClick(e)}>
-        <Icon type='fa-arrow-left' style={styles.icon} />
-        <Text type='BodyPrimaryLink' onClick={e => this.onClick(e)}>Back</Text>
+        <Icon type='fa-arrow-left' style={{...styles.icon, ...this.props.iconStyle}} />
+        {!this.props.iconOnly && <Text type='BodyPrimaryLink' style={this.props.textStyle} onClick={e => this.onClick(e)}>Back</Text>}
```

Comment:
i think you're out of date. see https://github.com/keybase/client/blob/master/shared/common-adapters/back-button.desktop.js
i exposed the title, so we should just allow that to be null (not undefined) so you can have this. instead of a specific flag for icononly